website/docs/r/continuous_compliance_notification.html.markdown

```diff
@@ -45,57 +45,89 @@ resource "dome9_continuous_compliance_notification" "test_notification" {
 
 The following arguments are supported:
 
-* `name` - (Required) The cloud account id.
-* `change_detection` - (Required) The account number.
+* `name` - (Required) The cloud account id in Dome9.
+* `description` - (Optional) Description of the notification.
+* `alerts_console` - (Optional) send  findings (also) to the Dome9 web app alerts console (Boolean); default is False.
+
+at least one of `scheduled_report` or `change_detection` must be included
```

Comment:
if we have just alert_console there is no need for the rest so:
at least one of the `alerts_console`,`scheduled_report` or `change_detection` must be included